docs/activity_log.md

```diff
@@ -0,0 +1,10 @@
+# Activity log
+
+In order to make your component compatible with the activity log, you need to follow these steps:
+
+1. Make your model include the `Decidim::Traceable` module. This will enable Decidim to create versions every time your model records are changed. It uses [`paper_trail`](https://github.com/airblade/paper_trail) to generate the versions.
```

Comment:
Is this really mandatory or recommended?

Reply:
It will be needed because the Product team wants to see all changes in all fields when the user updates a resource. For this, the best way is to have `paper_trail` included though the `Traceable` module

Reply:
Then maybe we need some kind of validation?

Reply:
Done, logs are not saved if the resource is not traceable.